src/server_manager/scripts/fill_packaging_opts.sh

```diff
@@ -23,15 +23,15 @@
 # script, allowing fill_packaging_opts.sh to fill variables for the caller.
 
 # Input: "/absolute/path/src/server_manager/electron_app/something_action.sh"
-# Output: "yarn do server_manager/electron_app/something"
+# Output: "npm run action server_manager/electron_app/something"
 readonly ELECTRON_PATH='server_manager/electron_app/'
 readonly RELATIVE="${1#*/src/${ELECTRON_PATH}}"
-readonly YARN_COMMAND="yarn do ${ELECTRON_PATH}${RELATIVE%_action.sh}"
+readonly NPM_COMMAND="npm run action ${ELECTRON_PATH}${RELATIVE%_action.sh}"
```

Comment:
```suggestion
readonly NPM_COMMAND="npm run action ${ELECTRON_PATH}${RELATIVE%.action.sh}"
```

You might want to confirm that this works.

Reply:
Was able to run the `server_manager/electron_app/build` action, which calls this script